db/view/view.cc

```diff
@@ -188,6 +188,16 @@ db::view::base_info_ptr view_info::make_base_dependent_view_info(const schema& b
     std::vector<column_id> base_regular_columns_in_view_pk;
     std::vector<column_id> base_static_columns_in_view_pk;
```

Comment:
Nit: typo in the second commit's description: `coressponding` -> `corresponding`